lib/src/main/java/com/auth0/jwt/JWTVerifier.java

```diff
@@ -452,7 +453,13 @@ private void assertDateIsPast(Date date, long leeway, Date today) {
 
     private void assertValidAudienceClaim(List<String> audience, List<String> value) {
         if (audience == null || !audience.containsAll(value)) {
-            throw new InvalidClaimException("The Claim 'aud' value doesn't contain the required audience.");
+            throw new InvalidClaimException(String.format("The Claim '%s' value doesn't contain the required audience.", PublicClaims.AUDIENCE));
+        }
+    }
+
+    private void assertValidIssuerClaim(String issuer, List<String> value) {
+        if (issuer == null || !value.contains(issuer)) {
+            throw new InvalidClaimException(String.format("The Claim '%s' value doesn't match the required issuer.", PublicClaims.ISSUER));
```

Comment:
Given that this method is already dedicated to the issuer claim, calling String.format is redundant. Please rollback